util/okapi-curl

```diff
@@ -3,26 +3,56 @@
 # Edit the `.okapi` file in your home directory to contain something like:
 #
 #	OKAPI_URL=https://east-okapi.folio-dev.indexdata.com
-#	OKAPI_TENANT=reshare_east
 #	OKAPI_TOKEN=xxxxxxxx
 #
+# You obtain OKAPI_TOKEN with
+#
+#	okapi-curl login
+#
 # Now you can issue much simpler curl commands by using `okapi-curl`, for example:
 #
-#	okapi-curl http://localhost:9130/copycat/target-profiles
+#	okapi-curl /copycat/profiles
+# or
+#	okapi-curl /copycat/profiles -d"{}"
 
 if [ $# = 0 ]; then
-   echo "Usage: $0 <path> [<curl-options>]" >&2
-   exit 1;
+	echo "Usage: $0 <path> [<curl-options>]" >&2
+	exit 1;
+fi
+
+. ~/.okapi
+if test -z "$OKAPI_URL"; then
+	echo "OKAPI_URL must be given in .okapi"
+	exit 1
+fi
+if test "$1" = "login"; then
+	echo -n "Tenant:"
+	read tenant
```

Comment:
To me, prompting for user/password is obviously correct, but tenant is part of the identity of the service that you're logging into, so it should be already in the `~/.okapi` file. The process is
1. Set up URL and tenant in that file
2. Log in to the service that they define.

Reply:
No. You supply tenant, user and password. Rest is in token.